docs/code/EntryPoints.md

```diff
@@ -220,11 +220,11 @@ parameter.
 
 ## How to create an entry point for an existing ML.NET component
 
-1. Add a `LoadableClass` assembly attribute with the `SignatureEntryPointModule` signature as shown [here](https://github.com/dotnet/machinelearning/blob/9db16c85888e7163c671543faee6ba1f47015d68/src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LogisticRegression.cs#L27).  
+1. Add a `LoadableClass` assembly attribute with the `SignatureEntryPointModule` signature as shown [here](https://github.com/dotnet/machinelearning/blob/9db16c85888e7163c671543faee6ba1f47015d68/src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/LogisticRegression.cs#L27).  
```

Comment:
These links are now broken. The reason is that these links are using SHAs of a certain commit in the history of the repo. In that point in the history, this was named `ML.StandardLearners`.

So either you need to update the SHA in the link, or keep it `ML.StandardLearners`.

Reply:
Good catch. I have reverted this file so that we check-in stuff that is not broken. I will create another PR once this change is checked-in and I have SHA of the new commit that has these links in the new renamed directory.